README.md

```diff
@@ -1,134 +1,144 @@
+- Go in the directory `ota-ce-gen/devices/:uuid`, there we run the following command to connect aktualizr.
+> The aktualizr client is intended to be installed on devices that wish to receive OTA updates from an Uptane-compatible OTA server
+```
+sudo aktualizr --run-mode=once --config=config.toml
+```   
 
-Once your build is complete and the device (virtual or real) is running, you can generate device credentials, register them, and copy them to the device with `make new-client`. You'll need to specify a device name, the IP and SSH port of the device, and the IP address of the OTA CE instance. For example, if your OTA CE machine is running at 192.168.1.99 and you've built a Raspberry Pi image that's running at 192.168.1.50, you could do the following:
+---
+## Working with APIs
 
+>List all devices
 ```
-export DEVICE_ID=raspberrypi_1
-export DEVICE_ADDR=192.168.1.50
-export DEVICE_PORT=22
-export GATEWAY_ADDR=192.168.1.99
-make new-client
+curl deviceregistry.ota.ce/api/v1/devices
 ```
 
-This will also modify the hosts file on the client so it's able to connect.
-
-### Getting started with your own Kubernetes cluster
-
-You can start OTA CE with `make start-ota`. This will run OTA CE in the cluster
-configured in your `kubectl` config. However, OTA CE in its default
-configuration is intended to work with Minikube, so you will need to
-reconfigure it for your own environment. For example, `config/config.yaml:4`
-sets the hostname for configuring the ingress resources. Depending on your
-environment may need to make other changes.
-
-## Troubleshooting
-
-Try re-running `make start` if you receive any of the following errors:
+>Detailed information about a specific device
+```
+curl deviceregistry.ota.ce/api/v1/devices/<device-uuid>
+```
+>Packages updated on a specific device
+```
+curl director.ota.ce/api/v1/admin/devices/<device-uuid>
+```
 
-* `Unable to connect to the server: dial tcp 192.168.99.100:8443: getsockopt: operation timed out`
-* `ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)`
-* `error on line 12 of ota-community-edition/scripts/server.cnf
-140680409176832:error:0EFFF068:configuration file routines:CRYPTO_internal:variable has no valu
-e:conf/conf_def.c:563:line 12
-error: error reading ota.ce/server.chain.pem: no such file or directory`
+>Package update installation history of a specific device
+```
+curl deviceregistry.ota.ce/api/v1/devices/<device-uuid>/installation_history
+```
 
-If your device isn't connecting, verify that the virtualbox port forwarding of port 30443 is set up correctly.
+>Updates assigned to be installed on a specific device
+```
+curl director.ota.ce/api/v1/assignments/<device-uuid>
+```
 
+---
 ## License
 
 This code is licensed under the [Mozilla Public License 2.0](LICENSE), a copy of which can be found in this repository. All code is copyright [ATS Advanced Telematic Systems GmbH](https://www.advancedtelematic.com), 2016-2018.
```

Comment:
This license text is not at all accurate. The named company doesn't exist anymore, and it's not 2018 anymore.

Reply:
Sure, I did not realize it was not  accurate. I will remove it with a commit. Will you please direct me as to what would be accurate. Thanks.

Reply:
@tkfu do you know what's best to put here? My first thought was Here (who owns most of the original OTA CE), but if it's all new, then it shouldn't be licensed to them. Should we license it to Uptane?